src/ledger/ledger-db/Dockerfile

```diff
@@ -16,6 +16,16 @@ FROM postgres:16.3-alpine@sha256:de3d7b6e4b5b3fe899e997579d6dfe95a99539d154abe03
 # Need to get coreutils to get the date bash function working properly:
 RUN apk add --no-cache coreutils && rm -rf /var/cache/apk/*
 
+# Create a user and group with the same UID and GID as the postgress
+RUN if ! getent group postgres > /dev/null; then addgroup -S postgres; fi && \
+    if ! getent passwd postgres > /dev/null; then adduser -S postgres -G postgres; fi
```

Comment:
Container images should have deterministic build steps. Either these groups / bindings exist, or they don't. We should need conditional checks.

Reply:
removed the conditional check.